### ISSUE TITLE
Debugging & Tooling engine.getExchanges() is returning an error

### ISSUE THREAD

**Issue by benjamin-keil-crypto-worx** [benjamin-keil-crypto-worx/botfox#2]
Steps To Reproduce:

Follow [Debugging & Tooling](https://benjamin-keil-crypto-worx.github.io/botfox-wiki/#/pages/quickStart?id=debugging-amp-tooling) Tutoril on BitFox wiki

Add below Code 
![image](https://user-images.githubusercontent.com/123937063/230432701-94603834-b44f-4506-aa93-d6bbe8bed66c.png)

Expected Behavior:
It should print out all Exchanges

Actual Behavior: 

Error

````
C:\Users\benja\develop\bf-testing\bitfox-tests-debugging-tooling.js:28
    console.log(engine.getExchanges());
                       ^

TypeError: engine.getExchanges is not a function
    at C:\Users\benja\develop\bf-testing\bitfox-tests-debugging-tooling.js:28:24
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
````

**Comment by benjamin-keil-crypto-worx** [comment]
Needs an update to Documentation the getExchanges() is a static method and it should be called like this:

``require("bitfox").bitfox.BitFoxEngine.getExchanges()``

**Comment by benjamin-keil-crypto-worx** [comment]
Updated Wiki fix available in #6
